README.rst

```diff
@@ -201,7 +201,7 @@ I already have a couple of ideas for future versions:
 .. _WSGI application: http://www.python.org/dev/peps/pep-0333/
 .. _PyPI: http://pypi.python.org/pypi/pytest-localserver/
 .. _bleeding-edge archive: https://bitbucket.org/pytest-dev/pytest-localserver/get/tip.tar.gz
```

Comment:
We should replace this line with a link to the Github project page .

Reply:
Oooh good catch thanks

Reply:
Fixed in the latest push